lib/sentry/application.ex

```diff
@@ -19,11 +19,14 @@ defmodule Sentry.Application do
         []
       end
 
+    integrations_config = Keyword.fetch!(config, :integrations)
+
     children =
       [
         {Registry, keys: :unique, name: Sentry.Transport.SenderRegistry},
         Sentry.Dedupe,
-        Sentry.Integrations.CheckInIDMappings
+        {Sentry.Integrations.CheckInIDMappings,
+         [max_expected_check_in_time: integrations_config[:max_expected_check_in_time]]}
```

Comment:
We always expect this:

```suggestion
         [max_expected_check_in_time: Keyword.fetch!(integrations_config, :max_expected_check_in_time]}
```